### ISSUE TITLE
[🐞] qwik-react is not bundling types in version 2.0.0

### ISSUE THREAD

**Issue by daltongray** [BuilderIO/qwik#2370]
### Which component is affected?

Qwik React

### Describe the bug

- Just updated to qwik-react 2.0.0. 
- Typescript is failing to pick up the types for qwik-react. 
- Cannot use intellisence / TS build

When importing code like this: 
`import { qwikify$ } from "@builder.io/qwik-react";`

I get a TS Error like this:
`TS7016: Could not find a declaration file for module '@builder.io/qwik-react'.`

### Reproduction

https://github.com/daltongray/qwik-react-bug-repro

### Steps to reproduce

Run `npm create qwik@latest` then `npm run qwik add react`.

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (16) x64 AMD Ryzen 7 5800H with Radeon Graphics
    Memory: 8.43 GB / 27.86 GB
  Binaries:
    Node: 16.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
    npm: 7.24.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (107.0.1418.62)
    Internet Explorer: 11.0.22000.120
  npmPackages:
    @builder.io/qwik: 0.15.1 => 0.15.1
    @builder.io/qwik-city: 0.0.128 => 0.0.128
    @builder.io/qwik-react: 0.2.0 => 0.2.0
    vite: 3.2.4 => 3.2.4
```

### Additional Information

Looking at the code, I thing the qwik-react package.json file needs to change in the following way: 
`line:68` `  "types": "./lib/types/index.d.ts"` => `"types": "./lib/types/index.qwik.d.ts"`

**Comment by manucorporat** [comment]
Just released 0.2.1! can you try updating?

**Comment by daltongray** [comment]
@manucorporat - types issue is fixed! Thank you!
